teuthology/task/install.py

```diff
@@ -760,6 +760,8 @@ def rh_install_pkgs(ctx, remote, installed_version):
     """
     pkgs = ['ceph-deploy']
     rh_version_check = {'0.94.1': '1.3.0', '0.94.3': '1.3.1'}
+    log.info("Remove any epel packages installed on node %s", remote.shortname)
+    remote.run(args=['sudo', 'yum', 'remove', run.Raw("leveldb xmlstarlet fcgi"), '-y'],check_status=False)
```

Comment:
Why is this being removed if the PR it depends upon is adding it specifically for RHEL?